GETTING_STARTED.md

```diff
@@ -0,0 +1,380 @@
+ * [1. Preface](#1-preface)
+    * [1.1. Information is widely spread across the company](#11-information-is-widely-spread-across-the-company)
+    * [1.2. Connecting the dots](#12-connecting-the-dots)
+    * [1.3. Editing / Publishing Separation](#13-editing--publishing-separation)
+ * [2. Getting started](#2-getting-started)
+    * [2.1. Data retrieval](#21-data-retrieval)
+    * [2.2. Data types &amp; column kinds](#22-data-types--column-kinds)
+       * [Primitive data types](#primitive-data-types)
+          * [Examples](#examples)
+       * [Higher data types](#higher-data-types)
+          * [<code>multi-language</code> and <code>multi-country</code>](#multi-language-and-multi-country)
+          * [<code>link</code>](#link)
+          * [<code>concat</code>, and <code>group</code>](#concat-and-group)
+          * [<code>attachment</code>](#attachment)
+
+
+# 1. Preface
+
+GRUD is an acronym and stands for "generic relational enterprise database". The following document should give a short introduction to GRUD and its principles.
+
+## 1.1. Information is widely spread across the company
+In today's companies data and information is widely spread across divisions and departments. We saw the need for storing Excel-like data in a structured and centralized way. A company could and should take advantage of connecting all this widespread data.
+
+## 1.2. Connecting the dots
+One of the main ideas is to store enterprise data in simple tables which are interconnected — which do have relations. You could start with simple and dumb tables. If you start to connect the data in the right way, you will leverage from rich product information. This idea is not new, but every relational database (Oracle, PostgreSQL, MySQL, etc.) out there shows how powerful it can be. We took this idea and its simple core concepts and built a user & consumer centered enterprise database. The core parts of GRUD consists of an RESTful API and an easy-to-use web-based user interface.
+
+## 1.3. Editing / Publishing Separation
+Another driving force behind GRUD's software architecture is the principle "Separation of Concerns". Traditional content management systems ship both "content editing" & "content publishing" in one piece of software. Both systems are interleaved but still serve highly different use cases. Content editing and content publishing in an omnichannel universe do have totally different requirements. The life-cycle of e.g. a website is different to a central enterprise database like GRUD.
+
+# 2. Getting started
+
+> All the following URLs are relative to your API endpoint.
+> 
+> E.g. https://example.org/api
+
+## 2.1. Data retrieval
+
+The schema is structured in tables and columns. A table consists of a set of columns and of course its rows. All API endpoints are generic and therefore are not bound to your schema at all. The next three examples will show you how to traverse the data structure.
```

Comment:
I would expect a little more information about the general organisation of data inside GRUD. I see that details are coming later, but I think the reader needs to understand the idea behind the data structure before being told how to request details. Something like GETing/POSTing to the endpoints will deliver/change the following:

- /table/[n]
  - table data & metadata (visibility, grouping,...)
- columns/[n]
  - information how to process/understand data from rows/cells (kinds, groupings, data format,...)
- /rows/[n]
  - raw, structured content of cells in that row
  - metadata for rows and their cells (flags, annotations)

and that to process a table, information should be requested in that order. Knowing that in the beginning would greatly have helped me getting started.